lemur/certificates/views.py

```diff
@@ -817,6 +819,7 @@ def get(self, certificate_id):
         return service.get(certificate_id)
 
     @validate_schema(certificate_edit_input_schema, certificate_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
this restriction doesn't make much sense to me. I don't understand the risk of a user updating notifications / ownership of a cert they already own

Reply:
same as Line 546